extensions/screenocr/src/utils.ts

```diff
@@ -1,54 +1,26 @@
-import { environment } from "@raycast/api";
-import { execa, ExecaError } from "execa";
-import { chmod } from "fs/promises";
-import { join } from "path";
+import { recognizeText as recognizeTextSwift } from "swift:../swift";
 import { getUserSelectedLanguages, usePreferences } from "./hooks";
 
 const recognizeText = async (isFullScreen = false) => {
   const preference = usePreferences();
-  const command = join(environment.assetsPath, "recognize-text");
-  await chmod(command, "755");
 
   try {
     const languages = await getUserSelectedLanguages();
 
-    const args: string[] = [];
-
-    if (isFullScreen) {
-      args.push("--fullscreen");
-    }
-
-    if (preference.languageCorrection) {
-      args.push("--languagecorrection");
-    }
-
-    if (preference.ocrMode === "fast") {
-      args.push("--fast");
-    }
-
-    if (preference.ignoreLineBreaks) {
-      args.push("--ignorelinebreaks");
-    }
-
-    if (preference.keepImage) {
-      args.push("--keepImage");
-    }
-
-    if (preference.customWordsList) {
-      args.push("--customwordslist", preference.customWordsList);
-    }
-
-    args.push("--languages");
-    args.push(languages.map((lang) => lang.value).join(" "));
-
-    const { stdout } = await execa(command, args);
-    return stdout;
+    const recognizedText = await recognizeTextSwift(
```

Comment:
This is bad pattern when you have more than 2 args, use the previous code style of flags.

Reply:
Here's how you can review this PR: https://developers.raycast.com/basics/review-pullrequest

Send me how you want the code to look, but use the new macros.